src/components/createTestCase/CreateTestCase.tsx

```diff
@@ -132,6 +133,17 @@ const INITIAL_VALUES = {
   groupPopulations: [],
 } as TestCase;
 
+interface PropTypes {
+  isActive?: boolean;
+}
+
+const MenuItemContainer = tw.ul`bg-transparent flex px-8`;
+const MenuItem = styled.li((props: PropTypes) => [
+  tw`relative h-2 mr-3 ml-3 mt-3 mb-0 not-italic font-rubik font-medium text-sm leading-4 text-gray`,
```

Comment:
does this `text-grey` value the same as the one in design mockup #515151?

Reply:
Thank you, I just changed it